src/foam/layout/Section.js

```diff
@@ -54,6 +54,9 @@ foam.CLASS({
       // Accepts function and string
       name: 'subTitle'
     },
+    {
+      name: 'wizardTitle'
+    },
```

Comment:
Adding this expression might avoid the need to set wizardletTitle in some cases. (this is already handled by the expression in `WizardletSection`, but I just realized it's better over here in case something else uses this property in the future)
```
{
  name: 'wizardTitle',
  expression: function (title) {
    return title;
  }
},
```

Reply:
Yeah, I agree. There might be cases where you may want a separate title on the nav, and others may be same.